convert2rhel/unit_tests/rhnclassic_test.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
```

Comment:
It's better to keep names of test files match the files they're testing in, so rename this to `rhelvariant_test.py`

Reply:
I agree, currently the test files are named after the code files. I'd stick to that unless we have a good reason for not to.